client/scripts/views/pages/new_proposal/new_proposal_form.ts

```diff
@@ -980,15 +977,11 @@ const NewProposalForm = {
             ]),
             m(FormGroup, [
               m(FormLabel, 'Proposal Executor'),
-              m(DropdownFormField, {
-                choices: (app.chain as Aave).governance.api.Executors.map(
-                  (r) => ({ name: 'executor', value: r.address, label: `${r.delay / (60 * 60 * 24)} Day(s) Voting` })
-                ),
-                callback: (result) => {
-                  vnode.state.executor = result;
```

Comment:
@jnaviask I am not sure why the two different dropdowns mutates the same state `vnode.state.executor`

Reply:
Because they refer to the same thing!